bin/runtime/src/lib.rs

```diff
@@ -300,6 +306,8 @@ impl pallet_scheduler::Config for Runtime {
     type MaxScheduledPerBlock = MaxScheduledPerBlock;
     type WeightInfo = pallet_scheduler::weights::SubstrateWeight<Runtime>;
     type OriginPrivilegeCmp = EqualPrivilegeOnly;
+    type PreimageProvider = Preimage;
```

Comment:
If possible use `()` instead of `Preimage` and write a proposal of adding this pallet to our runtime on some channel.

Reply:
Done, it works! The preimage task is tracked [here](https://cardinal-cryptography.atlassian.net/browse/AZ-748)